app/src/main/kotlin/org/cru/godtools/ui/about/AboutActivity.kt

```diff
@@ -22,7 +22,7 @@ class AboutActivity : BasePlatformActivity<ActivityGenericComposeWithNavDrawerBi
     // region Lifecycle
     override fun onBindingChanged() {
         super.onBindingChanged()
-        binding.genericActivity.frame.setContent { AboutLayout() }
+       binding.genericActivity.frame.setContent { AboutLayout() }
```

Comment:
I know you are still relatively new to git/github, just pointing this out to hopefully help you mature your git/github practices.

Random code churn like this can lead to several different issues:
1. merge conflicts if your branch accidentally updates a line that another PR also updates (intentionally or not)
2. more complicated git history/git blame
   * When looking at the git history for `AboutActivity` it would now include commits from this PR which is completed unrelated to the AboutActivity
3. Less focused PRs for reviews, instead of being able to focus on just what changed in the PR I now have a bunch of random noise I need to filter over to see if anything functionally changed with that updated code

Reply:
how I personally avoid this happening is on the command line I use the git staging area to build out my commit before committing it. I use `git add filename.ext` and `git add -p filename.ext` to add files to the staging area, `git reset filename.ext` to remove a file from the staging area, and `git commit` when I'm satisfied with committing the changes I have staged.

It looks like Github Desktop supports selecting individual changes and files when creating a commit as well, in the commit UI, you can use checkboxes to select/deselect files you want to include in a commit. And when previewing the changes in a file you can select/deselect individual lines that you want to include in the commit.